posthog/batch_exports/service.py

```diff
@@ -99,6 +100,7 @@ class S3BatchExportInputs:
     kms_key_id: str | None = None
     endpoint_url: str | None = None
     file_format: str = "JSONLines"
+    max_file_size_mb: int | None = None
```

Comment:
thought it would be better to have the user-facing parameter in MB rather than bytes to make it more user-friendly

Reply:
Makes sense :+1: